terraform/database.tf

```diff
@@ -12,18 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-resource "random_string" "db-name" {
-  length  = 5
-  special = false
-  number  = false
-  upper   = false
-}
-
 resource "google_sql_database_instance" "db-inst" {
   project          = data.google_project.project.project_id
   region           = var.db_location
   database_version = "POSTGRES_11"
-  name             = "en-${random_string.db-name.result}"
+  name             = var.db_name
```

Comment:
Note that "name" is ignored on changes, so this won't cause a re-create

Reply:
Is that documented anywhere? Didn't see it in https://www.terraform.io/docs/providers/google/r/sql_database_instance.html#argument-reference